javascript/javascript2/week1/hyfBayHelpers.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function() {
```

Comment:
I'm not sure why this function is assigned to `window`. Can you assign it as a global function?
```suggestion
function getAvailableProducts() {
```